### PR TITLE
Line 81 rectified and grammatical mistakes cleared

### PR DESCRIPTION
**Pull request by nandanasreeraj123**
This PR rectifies issue #1359 
Grammatical errors were also modified.

- [x]  on line 81, fix the example from using this.state to this.setState.
- [x]  Correct any grammar mistakes
- [x]  Correct any md mistakes
- [x]  Correct any code mistakes

### CODE REVIEW

File: react-js/react-part-2.md [hunk 1]
```diff
@@ -108,12 +110,12 @@ Now the problem arises: How do we store and change component information we want
       message: 'new message'
     });
     ```
-- The important thing to remember about `setState` is that is causes the component to re-render. Well, in introduction to React, we have learnt that React follows a special path called the **React cycle**. So, when state is updated using `setState`, React goes into the **React cycle** and then the changes are compared from the previous **React virtual DOM element**, changes are done accordingly and those changes are reflected in the UI.
+- The important thing to remember about `setState` is that it causes the component to re-render. Well, in introduction to React, we have learnt that React follows a special path called the **React cycle**. So, when state is updated using `setState`, React goes into the **React cycle** and then the changes are compared to the previous **React virtual DOM element**. The changes are made accordingly and are reflected in the UI.
```
Comment (Kundan28) [inline react-js/react-part-2.md:113]:
```suggestion
- The important thing to remember about `setState` is that it causes the component to re-render. Well, in introduction to React, we have learnt that React follows a special path called the **React cycle**. So, when state is updated using `setState`, React goes into the **React cycle** and then the changes are compared from the previous **React virtual DOM element** and the changes are made accordingly and are reflected in the UI.
```

File: react-js/react-part-2.md [hunk 2]
```diff
@@ -130,13 +132,13 @@ constructor(){
...
 
-3. A simple tag to display the number i.e. value defined in state.
+3. Create a simple tag to display the number i.e. the value defined in state.
 
 ```html
 <p>{this.state.value}</p>
 ```
 
-4. That is all which will be displayed on the page. Now we need to implement the functionality when the buttons are clicked.
+4. That is all what will be displayed on the page. Now we need to implement the functionality when the buttons are clicked.
```
Comment (Kundan28) [inline react-js/react-part-2.md:141]:
```suggestion
4. That is all which will be displayed on the page. Now we need to implement the functionality when the buttons are clicked.
```

File: react-js/react-part-2.md [hunk 3]
```diff
@@ -308,4 +310,4 @@ UI should contain -
 
 - [How setState works](https://css-tricks.com/understanding-react-setstate/) - In depth knowledge of reconcilation process of React.
 - [Update state](https://medium.com/@baphemot/understanding-reactjs-setstate-a4640451865b) - State can be updated in two ways, either by passing an object or by passing function as a parameter.
-- [State in functional component](https://reactjs.org/docs/hooks-state.html) - So far we have seen how to use state in class components. State can be defined in functional components as well by using additional package called Hooks.
+- [State in functional component](https://reactjs.org/docs/hooks-state.html) - So far, We have seen how to use state in class components. State can be defined in functional components as well by using additional package called Hooks.
```
Comment (Kundan28) [inline react-js/react-part-2.md:313]:
```suggestion
- [State in functional component](https://reactjs.org/docs/hooks-state.html) - So far, we have seen how to use state in class components. State can be defined in functional components as well by using additional package called Hooks.
```

**Comment by nandanasreeraj123** [comment]
changes have been made!

**Comment by Kundan28** [comment]
Updated in scoresheet too.

**Comment by nandanasreeraj123** [comment]
Thanku

**Pull request merged**